lib/cmsis_rtos_v1/cmsis_wait.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel_structs.h>
+#include <cmsis_os.h>
+
+/**
+ * @brief Wait for Timeout (Time Delay).
+ *
+ * Parameters
+ * [in]	millisec - "time delay" value
+ *
+ * Returns
+ * Status code that indicates the execution status of the function.
+ */
+osStatus osDelay(uint32_t delay_ms)
+{
+	if (_is_in_isr()) {
+		return osErrorISR;
+	}
+
+	k_sleep(delay_ms);
+	return osEventTimeout;
```

Comment:
Another whole file just for a tiny wrapper.  Fold with something else.

Reply:
This may have at least one more function coming up. Will take a decision later depending upon how big that is going to be.